sample/full/cmd/flags.go

```diff
@@ -50,6 +51,7 @@ func init() {
 			fmt.Println("uint     ", c.Flags.Uint("uint"))
 			fmt.Println("uint64   ", c.Flags.Uint64("uint64"))
 			fmt.Println("float    ", c.Flags.Float64("float"))
+			fmt.Println("StringSLice",c.Flags.StringSlice("Headers"))
```

Comment:
Please fix typo.

Reply:
and make sure to run go fmt, there is a missing whitespace between the arguments

Reply:
done